io_scene_godot/converters/mesh.py

```diff
@@ -573,6 +543,47 @@ def get_tup(self):
 
         return tup
 
+    @staticmethod
```

Comment:
If you use `@classmethod`, then the first parameter (typically called `cls` is a reference to the uninstantiated class. This means you can do `new_vert = cls()`